ReactWindows/ReactNative/Modules/AccessibilityInfo/AccessibilityInfoModule.cs

```diff
@@ -0,0 +1,78 @@
+﻿using ReactNative.Bridge;
+using ReactNative.Modules.Core;
+using System.Collections.Generic;
+using Windows.UI.ViewManagement;
+
+namespace ReactNative.Modules.Accessibilityinfo
+{
+    class AccessibilityInfoModule : ReactContextNativeModuleBase
+    {
+        private readonly AccessibilitySettings _accessibility = new AccessibilitySettings();
+        private readonly UISettings _settings = new UISettings();
+
+        private string GetRgbaString(UIElementType type)
+        {
+            var color = _settings.UIElementColor(type);
+            return "rgba(" + color.R + "," + color.G + "," + color.B + "," + color.A + ")";
+        }
+
+        // TODO: These are the 8 high contrast colors in Windows. Eventually they will be exposed to JS.
```

Comment:
can you file an issue for this TODO item?

Reply:
Done: https://github.com/Microsoft/react-native-windows/issues/1459